apps/elewa-group-website/src/_variables.scss

```diff
@@ -15,6 +15,13 @@
   --elewa-group-website-dmsans-medium-italic: 'DMSans-Medium-Italic';
   --elewa-group-website-dmsans-regular: 'DMSans-Regular';
 
+  // elewa-hero
+  --elewa-group-website-font-color: #ffff;
```

Comment:
rename this to --elewa-group-website-color-white

to avoid redundancy of color variables